### PR TITLE
add js,css,localization

### PR DESCRIPTION
**Pull request by kienpham9819**

### CODE REVIEW

**Comment by kienpham9819** [comment]
ready

File: resources/css/mycss.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.error {
+	color: red;
+	margin-left: 10px;
```
Comment (HoaiPEC) [inline resources/css/mycss.css:3]: indent 4 space

File: resources/css/mycss.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+}
+
+.btnD {
+	color: red;
```
Comment (HoaiPEC) [inline resources/css/mycss.css:7]: tuong tu

File: resources/lang/vi/error.php [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?php 
+
+return [
+	"required" => "Không được để trống trường này",
+	"over_leng" => "Không được vượt quá :value kí tự ",
+	"unique" => "Tên này đã tồn tại !",
+	"be_image" => "Bắt buộc phải là ảnh",
+	"image_end" => "Chỉ cho phép kiểu đuôi jpg,png,jpeg,gif,svg",
+	"image_size" => "Kích thước ảnh đã vượt quá 100kb",
```
Comment (HoaiPEC) [inline resources/lang/vi/error.php:9]: indent 4 space

File: resources/lang/en/title.php [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+	"birthday" => "Birthday",
+	"gender" =>"Gender",
+	"male" => "Male",
+	"female" =>"Female",
+	"avatar" => "Avatar",
+	"work_at" => "Work at",
+	"close" => "Close",
+	"noti_add" => "You added success",
+	"noti_edit" => "You editted success",
+	"noti_delete" => "You deleted success",
```
Comment (HoaiPEC) [inline resources/lang/en/title.php:34]: tuong tu

File: resources/lang/vi/title.php [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+	"male" => "Nam",
+	"female" =>"Nữ",
+	"avatar" => "Ảnh đại diện",
+	"work_at" => "Làm việc tại",
+	"close" => "Đóng",
+	"noti_add" => "Bạn đã thêm thành công",
+	"noti_edit" => "Bạn đã sửa thành công",
+	"noti_delete" => "Bạn đã xóa thành công",
+ ];
+
```
Comment (HoaiPEC) [inline resources/lang/vi/title.php:36]: end of file

File: config/lang.php [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<?php 
+return [
+	'locale' => [
+		'en',
+		'vi',
+	],
+];
```
Comment (huyenttm-0901) [inline config/lang.php:7]: indent 4 spaces

File: config/linkimage.php [hunk 7]
```diff
@@ -0,0 +1,7 @@
+<?php 
+
+return [
+	'destination' => 'public/images/staffs',
+	'sourceurl' => 'storage/images/staffs/',
+	'defaulturl' => 'https://sun-asterisk.vn/wp-content/uploads/2019/03/Sun-Logotype-RGB-01.png',
+];
```
Comment (huyenttm-0901) [inline config/linkimage.php:7]: tương tự

File: resources/lang/vi/title.php [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+	"gender" =>"Giới tính",
+	"male" => "Nam",
+	"female" =>"Nữ",
+	"avatar" => "Ảnh đại diện",
+	"work_at" => "Làm việc tại",
+	"close" => "Đóng",
+	"noti_add" => "Bạn đã thêm thành công",
+	"noti_edit" => "Bạn đã sửa thành công",
+	"noti_delete" => "Bạn đã xóa thành công",
+ ];
```
Comment (huyenttm-0901) [inline resources/lang/vi/title.php:35]: eof

**Comment by kienpham9819** [comment]
ready

**Review by huyenttm-0901 — COMMENTED** [review]
(merged)

**Pull request merged**